### DISCUSSION TITLE
The launch.json file is automatically displayed when debugging C language

### DISCUSSION THREAD

**Discussion by 123lujiaqiang** [coder/code-server#4637] [Q&A]
My code-server is deployed using Docker, the container environment is Ubuntu, various environments and plug-ins exist, but when debugging C and C ++ language, always enter the launch.json file (F5-- >; Select environment C/C++-- >; Json) but I've configured it and I don't have any problems. I hope to get answers

**Comment by jsjoeio** [comment]
Does [this](https://github.com/cdr/code-server/issues/4221) sound like your issue?

**Reply by 123lujiaqiang** [reply]
Yeah, that's the problem I had, maybe I didn't describe it accurately enough. The code server cannot debug C and C++ repeatedly. Every time I click on Run and debug, it will bring up the select environment (C++(GDB/LLDB)) and it will jump to the launch.json file, but I have configured the launch.json and tasks.

**Reply by 123lujiaqiang** [reply]
Yes, this issue is my problem

**Reply by jsjoeio** [reply]
We're hoping this might be fixed in the next release so keep an eye out!

**Reply by 123lujiaqiang** [reply]
OK, thank you
